translate/convert/po2web2py.py

```diff
@@ -36,7 +39,7 @@ def __init__(self):
         return
 
     def convertstore(self, inputstore, includefuzzy):
-        str_obj = BytesIO()
+        str_obj = StringIO()
```

Comment:
Switching to a StringIO was the only way I was able to get both Python 2 and 3 compatibility with support for unicode characters (at least whilst producing consistent, usable results).  I'm open to suggestions if someone has a better idea on how to tackle this, though?

Reply:
Is the StringIO approach allowing the code to handle non-ASCII translations?  It is not currently tested, so it is hard to be sure.

Reply:
On Python 2 it didn't seem to matter, but on Python 3, yes, it was my non-ASCII encoding solution.

Reply:
I didn't check yet, but if we don't have tests for those scenarios it would be great if you can add some.

Reply:
If it is not possible to make this work for Python 2 I am open to make this converter a Python 3 only since we are less than one year away from Python 2 being abandoned. In the past we had converters that worked only with Python 2, and this is not one of the most widely used converters, so IMHO it shouldn't such a big issue. Opinions?